tests/rb_json_test.c

```diff
@@ -69,12 +68,12 @@ static void rb_assert_json_value(const struct checkdata_value *chk_value,const j
 	if(chk_value->value == NULL && json_value != NULL) {
 		fprintf(stderr,"Json key %s with value %s, should not exists in (%s)\n",
```

Comment:
fprintf + assert_true(false) => fail_msg("This is some error message for test");